app/controllers/api/members_controller.rb

```diff
@@ -19,10 +19,23 @@ def forget
     head :no_content
   end
 
+  def update
+    # Only used to update AKID after a new member has been created. Called by the ActionKit worker.
+    @member = Member.find_by_id(permitted_params[:id])
+    if @member.blank?
+      render(json: { errors: "Member with ID #{permitted_params[:id]} not found" }, status: :not_found) && return
+    end
+    if @member.update_attributes(actionkit_user_id: permitted_params[:akid])
+      render json: { member: @member }
+    else
+      render json: { errors: "Failure updating AKID on Member with ID #{permitted_params[:id]}", status: 422 }
```

Comment:
For the status, do we prefer symbols or codes? I like the `status: :not_found` you used above, so this could also be more readable with `status: :unprocessable_entity`, although I'm not particularly bothered by either.

Reply:
I'll make it a symbol :)